docs/api/session.md

```diff
@@ -28,6 +28,7 @@ The `session` module has the following methods:
 * `partition` string
 * `options` Object (optional)
   * `cache` boolean - Whether to enable cache.
+  * `quota` number - Use a quota size for a partition.
```

Comment:
define if quota is in bytes/megabytes etc.

Reply:
Done.